metricbeat/module/kubernetes/pod/pod.go

```diff
@@ -44,7 +43,7 @@ var (
 // init registers the MetricSet with the central registry.
 // The New method will be called after the setup of the module and before starting to fetch data
 func init() {
-	mb.Registry.MustAddMetricSet("kubernetes", "pod", New,
+	mb.Registry.MustAddMetricSet("kubernetes", util.PodResource, New,
```

Comment:
That change (and similar) are irrelevant to what this PR's title indicate. Shall we change the PR tittle or split the PR into smaller ones?